File: tests/MIXED/Both_Fog/both.c
void runatfog(char *s);

int count = 1;

jasync worker(int x) {

	jsleep(500);
	printf("Message at worker %d\n", x);
	runatfog("hello");
	count++;
}


int main(int argc, char *argv[]) {

    worker(1);
    return 0;
}

File: tests/Ver_1_Tests/MIXED/Both_Fog/both.js
jcond {
    fogonly: jsys.type == "fog";
    devonly: jsys.type == "device";
}

var count = 1;

jasync {fogonly} function runatfog(q) {

    console.log("At fog only...", q);
    runatdev(count);
    count++;
}

jasync {devonly} function runatdev(q) {
    console.log("At dev only..", q);
    worker(q);
}

